lang/ilias_de.lang

```diff
@@ -14,6 +14,21 @@
 // to the corresponding mode!
 // Language file names refer to ISO 639, see: http://www.oasis-open.org/cover/iso639a.html
 <!-- language file start -->
+changelog#:#delete_old_logs_title#:#Alte Changelog-Einträge löschen
+changelog#:#delete_old_logs_info#:#Entfernt Changelog-Einträge, welche älter als X Tage sind, aus der Datenbank.
```

Comment:
It would be great to have "X" as a variable that will be filled by ILIAS depending on the selected (or fixed) setting for it. The lang variable in the current version is not reqlly helpful for users as they have no idea about X